### PR TITLE
Allow overide ferment pip package

### PR DESCRIPTION
**Pull request by thiagotalma**
The project is not getting the care it deserves and it would be interesting to be able to inform another repository to install the ferment.

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -312,6 +312,11 @@ docker__pki_key: 'default.key'
 docker__ferment: True
 
                                                                    # ]]]
+# .. envvar:: docker__ferment_pip_package [[[
+#
+# Packages to install ferment from PyPI using the ``pip`` command.
+docker__ferment_pip_package: 'ferment'
+                                                                   # ]]]
```
Comment (drybjed) [inline defaults/main.yml]: Can you add a separate line here to have better spacing between the variables?

**Review by drybjed — APPROVED** [review]
Sounds good. I presume that instead of a name you can specify a git repository URL which `pip` will understand?

**Comment by thiagotalma** [comment]
Exact!
This is valid: `pip install https://github.com/diefans/ferment/archive/master.zip`

**Comment by thiagotalma** [comment]
thanks!
@drybjed, Can I update only this role with debops-update?

**Comment by drybjed** [comment]
@thiagotalma No, but you can cd into `~/.local/share/debops/debops-playbooks/roles/debops.docker/` and run `git pull`, this should update the repository of this role.

**Pull request merged**